syntax = "proto3";
package Protocol;

import "Enum.proto";
import "Struct.proto";

message EnterChatRoomReq
{
	int32 message = 1;
	int32 value = 2;
}

message EnterChatRoomAck
{
	uint32 id = 1;
}

message LoginReq
{
	string id = 1;
	string password = 2;
}

message LoginAck
{
	bool success = 1;
	string userID = 2;
}

message ChatRoomListReq
{
	bool success = 1;
}

message ChatRoomListAck
{
	bool success = 1;
	repeated ChatRoomInfo chatroomInfo = 2;
}

message CreateChatRoomReq
{

}

message CreateChatRoomAck
{

}

message RefreshChatRoomReq
{

}

message RefreshChatRoomAck
{

}